transmogrifier/sources/json/aardvark.py

```diff
@@ -417,6 +417,8 @@ def get_subjects(source_record: dict) -> list[timdex.Subject]:
             if key in source_record
         }.items():
             for subject in source_record[aardvark_subject_field]:
-                subjects.append(timdex.Subject(value=[subject], kind=kind_value))
+                subjects.append(  # noqa: PERF401
+                    timdex.Subject(value=[subject], kind=kind_value)
+                )
```

Comment:
I was digging into the `PERF401` linting ignore, and found myself still tripping a bit over this method.  

I think it's a sneaky one, as it's doing quite a bit: looping through possible field names, that may or may not have data, and setting an associated kind value if present.

Wanted to float an alternate approach that makes use of `.extend()` and `.get()` to avoid some checks, and allows for list comprehension.  Not a requested change, just a suggestion for an alternate form that I don't think requres linting ignores.  Hard to say if it's more or less readable.

```python
subjects = []

aardvark_subject_fields = {
    "dcat_keyword_sm": "DCAT; Keyword",
    "dcat_theme_sm": "DCAT; Theme",
    "dct_spatial_sm": "Dublin Core; Spatial",
    "dct_subject_sm": "Dublin Core; Subject",
    "gbl_resourceClass_sm": "Subject scheme not provided",
    "gbl_resourceType_sm": "Subject scheme not provided",
}

for subject_field, subject_kind in aardvark_subject_fields.items():
    subjects.extend(
        [
            timdex.Subject(value=[subject], kind=subject_kind)
            for subject in source_record.get(subject_field, [])
        ]
    )

return subjects
```

Reply:
This is much more readable, thanks!